### ISSUE TITLE
Suppress In Queue Call Recording

### ISSUE THREAD

**Issue by darbyje** [MyPureCloud/terraform-provider-genesyscloud#583]
Since the release of Call recording suppression during IVR and queue wait the api has been updated to include a parameter to toggle this state:
 "suppressInQueueCallRecording": boolean

This is not yet supported in the terraform provider, and my observation is that all queues created via terraform at this moment have this toggle defaulted to off

**Comment by charliecon** [comment]
No, this parameter hasn't been added yet. We'll get a dev working on this as soon as we can.

Thanks for bringing it to our attention.

(Tracking with DEVENGAGE-2045)

**Comment by charliecon** [comment]
Fix went out with version 1.24.0
